package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@angular/devkit-repo",
-  "version": "12.0.0-next.9",
+  "version": "12.0.0-rc.0",
```

Comment:
We are not moving to `rc` yet, we should still be doing a `next` release

Reply:
This is for the release next week :)

Reply:
Now that I have looked at the calendar... we are only doing feature freeze for one week before moving to rc.  My bad.